lib/Channel.js

```diff
@@ -215,7 +215,10 @@ class Channel {
     }
 
     sendMessage(body) {
-        return TwilioIPMessagingMessages.sendMessage(this.sid, body)
+        if(Platform.OS == 'android'){
+            this.initialize()
```

Comment:
@bradbumbalough it seems to work without waiting for the promise, perhaps because twilio handles these call synchronously...?  we don't specifically need anything from the initialize call, it just seems to set something up for further use of the channel... the warning I was getting said "Messaging lib not ready, retrying..." although maybe we should wait for the promise and set up error handling incase it gets rejected for any reason so it doesn't go into a loop...